src/backend/BSConfig.pm.template

```diff
@@ -238,4 +238,36 @@ our $servicedispatch = 1;
 #   repository_base => '',
 # };
 
+# our $container_registries = {
```

Comment:
Why are we not going to store that in the frontend?

We could even make this dynamic and store it in the database, no?